### DISCUSSION TITLE
What font does Spotify use?

### DISCUSSION THREAD

**Discussion by miademon3344** [community/community#40159] [Programming Help]
### Select Topic Area

Question

### Body

The font used by Spotify is Spotify Circular, a modified version of Gotham Circular, available in four weights and italics. We see this font on the Spotify website because the font was previously used in conjunction with Proxima Nova, which is similar to Gotham.

[Ezzee Fonts](https://ezzeefonts.com/)

**Comment by MuhammadYousufHere** [comment]
Seems to me they have their own version of fonts, 'spotify-circular'. you can find out more using google chrome extension 'what fonts'.

[What Fonts](https://chrome.google.com/webstore/detail/whatfont/jabopobgcpjmedljpbcaablpmlmfcogm?hl=en)

**Comment by WanYakuwea** [comment]
Hey there! If you're curious about the font that Spotify uses, it's actually called "Circular." It's a clean and modern font that suits Spotify's sleek and stylish aesthetic. By the way, have you checked out [songlifty.com](https://songlifty.com/spotify-promotion/)? It's an awesome platform that curates AI-generated songs, offering a unique listening experience. It's like diving into a whole new world of music!
